cmd/kube-controller-manager/app/plugins_providers.go

```diff
@@ -60,7 +61,7 @@ type pluginInfo struct {
 	pluginProbeFunction     probeFn
 }
 
-func appendAttachableLegacyProviderVolumes(allPlugins []volume.VolumePlugin, featureGate featuregate.FeatureGate) ([]volume.VolumePlugin, error) {
+func appendAttachableLegacyProviderVolumes(logger klog.Logger, allPlugins []volume.VolumePlugin, featureGate featuregate.FeatureGate) ([]volume.VolumePlugin, error) {
```

Comment:
CI failed: https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/116529/pull-kubernetes-typecheck/1635202025572536320
> FAILED   verify-typecheck-providerless.sh	108s

This function in file `cmd/kube-controller-manager/app/plugins_providerless.go` also needs to add the parameter logger `klog.Logger`

Reply:
Ah, that file has a `+build providerless` - that's why it's not usually built and I didn't see that failure earlier. `hack/verify-golangci-lint.sh` also doesn't find it. Interesting  - I wonder whether it should.

But that's an aside. I've fixed it locally and will push the fix in the next update.